lang/en/admin.php

```diff
@@ -1064,6 +1063,7 @@
 $string['proxyuser'] = 'Proxy username';
 $string['query'] = 'Query';
 $string['question'] = 'Question';
+$string['questionbanks'] = 'Question banks';
```

Comment:
As per discussed in the catch-up, we don't want to edit this file in this PR, so please leave as it is now.

Reply:
Rolled back